shell/server/installer.js

```diff
@@ -310,92 +310,74 @@ AppInstaller = class AppInstaller {
     };
 
     let protocol;
-    if (url.protocol === 'http:') {
-      protocol = Http;
-    } else if (url.protocol === 'https:') {
+    if (url.protocol === 'http:' || url.protocol === 'https:') {
+      protocol = Request.defaults({
+        maxRedirects: 1,
       // Since we will verify the download against a hash anyway, we don't need to verify the server's
       // certificate. In fact, the only reason we support HTTPS at all here is because some servers
       // refuse to serve over HTTP (which is, in general, a good thing). Skipping the certificate check
       // here is helpful in that it means we don't have to worry about having a reasonable list of trusted
       // CAs available to Sandstorm.
-      options.rejectUnauthorized = false;
-      protocol = Https;
+        strictSSL: false
+      });
     } else {
       throw new Error('Protocol not supported: ' + url.protocol);
     }
 
     // TODO(security):  It could arguably be a security problem that it's possible to probe the
```

Comment:
Hey I think you're fixing this TODO! (Networks with significant private internals should always specify a proxy.)